man/zeek.8

```diff
@@ -148,6 +150,28 @@ Output file for script execution statistics
 .TP
 .B ZEEK_DISABLE_ZEEKYGEN
 Disable Zeekygen (Broxygen) documentation support
+.SH OUTPUT FORMAT
+Output is written in multiple files depending on configuration. Default
+location is the current directory. Packets can be written to a tcpdump file.
+
+The output written by Zeek can be formatted in multiple ways using the
+logging framework.
+.PP
+The default are files in human-readable (ASCII) format and data is organized
+into columns (tab-delimited), They can be processed by the \fBzeek-cut\fR tool.
+
+
+.SH EXAMPLES
+Read a capture file:
+.br
+    # zeek -r test-capture.pcap
+.PP
+Usually Zeek is started by running \fBzeekctl\fR. To configure Zeek with an initial
+configuration, install, and restart:
```

Comment:
Perhaps "When running zeek on a live interface, it is usually started by running zeekctl"?

Also - I am not entirely sure I would go and directly mention zeekctl deploy here - since the default configuration will work close to nowhere - and if it randomly works, it will probably not do what is expected/wanted.

Reply:
I added a note that this default configuration may need to be updated.

From a few years experience with Zeek on OpenBSD, Debian and Ubuntu I have found that it works nicely by just changing the interface in node.cfg. I also think a part of this relies on the package maintainers.